stockfish/models.py

```diff
@@ -896,27 +896,17 @@ def convert_human_notation_to_sf_notation(self, move: str) -> str:
             if self.is_move_correct(move):
                 if (
                     isCapture
-                    and turnsInto != ""
-                    and self.get_what_is_on_square(dst) is None
-                ) or (
-                    isCapture
-                    and turnsInto == ""
                     and self.will_move_be_a_capture(move)
                     == Stockfish.Capture.NO_CAPTURE
                 ):
-                    raise ValueError("Move is not a capture")
+                    raise ValueError(f"{move} is not a capture.")
                 elif (
                     not isCapture
-                    and turnsInto != ""
-                    and self.get_what_is_on_square(dst) is not None
-                ) or (
-                    not isCapture
-                    and turnsInto == ""
                     and self.will_move_be_a_capture(move)
                     != Stockfish.Capture.NO_CAPTURE
                 ):
-                    print(
-                        "Warning: Move results in a capture, but capture was not indicated by the move string."
+                    raise ValueError(
+                        f"{move} results in a capture, but there isn't an 'x' indicating this in its notation."
```

Comment:
`move` at this point is the standard notation. this might confuse the user since that's not what they put in.

Reply:
Yup, noticed that when reading some of the test output. I've changed it to output a copy of the original move param now.